From 7b9cbc1f9e6b12d1c367eaac176c007d24f1ac81 Mon Sep 17 00:00:00 2001
From: Sven Joachim <svenjoac@gmx.de>
Date: Wed, 10 Jan 2024 17:31:53 +0100
Subject: [PATCH] Build-depend on dh-sequence-movetousr

This moves the systemd units into /usr/lib/systemd/system in trixie
and later, while keeping them in /lib/systemd/system in
bookworm-backports.
---
 debian/control | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/control b/debian/control
index 3729659..8d0cb4b 100644
--- a/debian/control
+++ b/debian/control
@@ -5,6 +5,7 @@ Build-Depends: debhelper-compat (= 13),
                bats,
                brz,
                dh-python,
+               dh-sequence-movetousr,
                fakeroot,
                git,
                python3:any,
--
2.43.0

